examples/mwaa/vpcendpoint.yaml

```diff
@@ -0,0 +1,215 @@
+# These endpoints are required if your subnets don't have a route
+# to a NAT Gateway.
+# See https://docs.aws.amazon.com/mwaa/latest/userguide/vpc-create.html
+# for details.
+---
+apiVersion: ec2.aws.crossplane.io/v1beta1
+kind: SecurityGroup
+metadata:
+  name: sample-mwaa-endpoint-sg
+spec:
+  deletionPolicy: Delete
+  forProvider:
+    description: Security Group for Amazon MWAA Environments to access VPC endpoints
+    groupName: mwaa-vpc-endpoints
+    ingress:
+      - ipProtocol: tcp
+        fromPort: 443
+        toPort: 443
+        ipRanges:
+          - cidrIp: 10.0.0.0/8
+            description: Sample CIDR
+    region: us-east-1
+    vpcId: <vpc-id>
```

Comment:
can we use here `vpcIdRef` ? - same for all other

Reply:
Fixed